kubectl-plugin/kubectl-waiops

```diff
@@ -880,6 +880,26 @@ else
     VERSION_AIOPSORCHESTRATOR=$(oc get $CSV_NAME -n $INSTALLATION_NAMESPACE -o jsonpath='{.spec.version}' | cut -d. -f1-2)
 fi
 
+# Get the subscription target version to detect pending upgrades
+if [ "${CLUSTER_SCOPE_INSTALL}" == "true" ]; then
+    SUB_NAMESPACE="openshift-operators"
+else
+    SUB_NAMESPACE="${INSTALLATION_NAMESPACE}"
+fi
+SUB_TARGET_VERSION=$(oc get subscription.operators.coreos.com -n ${SUB_NAMESPACE} ibm-aiops-orchestrator -o jsonpath='{.spec.currentCSV}' | grep -o '[0-9]\+\.[0-9]\+' || echo "")
```

Comment:
I don't think the name must always be "ibm-aiops-orchestrator" the subscription name can be anything, the package name target would be ibm-aiops-orchestrator.

Reply:
Good point. I added [9dd04dc](https://github.com/IBM/cp4waiops-samples/pull/318/commits/9dd04dc3d000ff6c689729442a47f3ca931ead3a) to cover this case.